pkg/scheduling/v1/scheduler.go

```diff
@@ -874,10 +876,12 @@ func (s *Scheduler) getSnapshotInput(mustSnapshot bool) (*SnapshotInput, bool) {
 				workerSlotUtilization[workerId] = &SlotUtilization{
 					UtilizedSlots:    0,
 					NonUtilizedSlots: 0,
+					TotalSlots:       0,
 				}
 			}
 
 			uniqueSlots[slot] = true
+			workerSlotUtilization[workerId].TotalSlots++
```

Comment:
Do we need to store total slots at all? Can we simply just pass the total of utilized + non-utilized to the prom metric?

Reply:
happy to remove this if that's good for you!